js/about/preferences.js

```diff
@@ -362,7 +380,10 @@ class BitcoinDashboard extends ImmutableComponent {
         : null
       }
       <div className='board'>
-        {this.coinbasePanel}
+        { (this.userInAmerica)
+          ? this.coinbasePanel
+          : this.exchangePanel
```

Comment:
i'm not sure, i just think it should show something reasonable if `exchangeUrl` is empty due to a network connection problem or something